### PR TITLE
Bag Totals, Bug Fixes, BinanceUS

### PR DESCRIPTION
**Pull request by HojouFotytu**
- created Monitor strategy shortcut "SOM" for new PT status: "config som enabled"
- fixed Monitor PBL sorting, broken by New PT status "config som enabled"
- Changed "Value" headers to "Cost" on Bag lists, since the value represents total cost, not current value.
- Added total cost and profit % on the bottom of dashboard bag list 
- Fixed broken links to PT wiki on Settings > Analyzer page.

### CODE REVIEW

File: Monitor/Pages/_get/DashboardTop.cshtml [hunk 1]
```diff
@@ -147,6 +147,11 @@
                 bool lostValue = false;
                 lostValue = (dcaLogEntry.TotalCost == 0.0) || (dcaLogEntry.AverageBuyPrice == 0.0);
 
+                // Aggregate totals
+                Model.TotalBagCost = Model.TotalBagCost + dcaLogEntry.TotalCost;
+                double TradingFee = (dcaLogEntry.Amount * dcaLogEntry.CurrentPrice) * 0.002;
```
Comment (djbadders) [inline Monitor/Pages/_get/DashboardTop.cshtml]: Are all exchanges charging a 0.2% fee?

**Comment by djbadders** [comment]
Look like a decent set of changes, nice work. One small comment to consider.

File: Monitor/Pages/_get/DashboardTop.cshtml (hunk 1 above)
Comment (HojouFotytu) [reply Monitor/Pages/_get/DashboardTop.cshtml]: - Updated with fees per/exchange.  Good catch.  ;)

**Comment by HojouFotytu** [comment]
- Added exchange: BinanceUS
- Added exchange name to "Market Trends" charts
- Added exchange name to Settings > General > Application
- Fixed code for fees per exchange

File: Monitor/Pages/_get/DashboardTop.cshtml [hunk 2]
```diff
@@ -149,7 +149,23 @@
...
+                }
+                if (Model.PTMagicConfiguration.GeneralSettings.Application.Exchange.Equals("BinanceUS")) {
+                  ExchangeFee = 0.002;
+                }
+                if (Model.PTMagicConfiguration.GeneralSettings.Application.Exchange.Equals("Bittrex")) {
+                  ExchangeFee = 0.0025;
+                }
+                if (Model.PTMagicConfiguration.GeneralSettings.Application.Exchange.Equals("Poloniex")) {
+                  ExchangeFee = 0.0025;
+                }
```
Comment (djbadders) [inline Monitor/Pages/_get/DashboardTop.cshtml]: Neater as a switch case block but will work :)

Comment (HojouFotytu) [reply Monitor/Pages/_get/DashboardTop.cshtml]: Good point.  Changed.

**Pull request merged**